packages/pharos/src/components/popover/pharos-popover.scss

```diff
@@ -0,0 +1,44 @@
+@use '../../utils/scss/mixins';
+
+:host {
+  display: block;
+  position: fixed;
+  z-index: 140;
+  pointer-events: none;
+  top: 0;
+  left: 0;
+}
+
+.popover {
+  @include mixins.interactive-focus;
+
+  background-color: var(--pharos-color-ui-10);
+  position: relative;
+  margin: 0;
+  list-style-type: none;
+  padding-left: 0;
+  padding-inline-start: 0;
+  box-shadow: 0 1px 2px rgb(18 18 18 / 0.3), 0 4px 8px 3px rgb(18 18 18 / 0.15);
```

Comment:
@lorilundy711 I think this is the first time we've introduced height to a component, is there any concern with the inconsistency here compared to e.g. dropdown menus?